src/features/projectsV2/ProjectsMap/InterventionDropDown/InterventionList.module.scss

```diff
@@ -15,9 +15,17 @@ $layoutPaddingSide: 20px;
   gap: 18px;
   position: absolute;
   z-index: 2;
-  top: calc($layoutPaddingTop + 70px);
   right: calc($layoutPaddingSide + 10px);
 }
+
+.dropdownButtonAlignmentAbove {
+  top: calc($layoutPaddingSide + 75px);
+}
+
+.dropdownButtonAlignmentBelow {
+  top: calc($layoutPaddingSide + 10px);
+}
+
```

Comment:
1. Is `layoutPaddingSide` used in these calculations by mistake?
2. Check the calculations, there is a small difference in position for the top of sites dropdown vs the top of the intervention type dropdown (when it is by itself).